README.md

```diff
@@ -161,10 +161,10 @@ These extensions will add syntax highlighting for `.browserslistrc` files.
     ]
   ```
 
-* If you want to change the default set of browsers, we recommend combining
-  `last 2 versions`, `not dead` with a usage number like `> 0.2%`. This is
+* If you want to change the default set of browsers, we recommend including
+  `last 2 versions and not dead > 0.2%`. This is
```

Comment:
I was actually unsure from the description if the intended combinator ist `and` or `or`

Reply:
`X, not dead` == `X and not dead`

I recommend using `last 2 versions, > 0.2%, not dead`